src/components/NewsletterSignup/NewsletterSignup.test.tsx

```diff
@@ -0,0 +1,297 @@
+import { render, screen } from "@testing-library/react";
+//import userEvent from "@testing-library/user-event";
+import { axe } from "jest-axe";
+import * as React from "react";
+import renderer from "react-test-renderer";
+
+import NewsletterSignup from "./NewsletterSignup";
+import { sectionDataMap } from "../../helpers/types";
+
+describe("NewsletterSignup Accessibility", () => {
+  const onSubmit = jest.fn();
+  it("Form state passes accessibility", async () => {
+    const { container } = render(
+      <NewsletterSignup
+        title="Testing"
+        onSubmit={onSubmit}
+        formHelperText={"Form helper."}
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("Error state passes accessibility", async () => {
+    const { container } = render(
+      <NewsletterSignup title="Testing" onSubmit={onSubmit} view={"error"} />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("Confirmation state passes accessibility", async () => {
+    const { container } = render(
+      <NewsletterSignup
+        title="Testing"
+        onSubmit={onSubmit}
+        view={"confirmation"}
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+
+  it("Bad email state passes accessibility", async () => {
+    const { container } = render(
+      <NewsletterSignup
+        title="Testing"
+        onSubmit={onSubmit}
+        view={"form"}
+        isInvalidEmail={true}
+        formHelperText={"That is not allowed."}
+      />
+    );
+    expect(await axe(container)).toHaveNoViolations();
+  });
+});
+
+describe("NewsletterSignup Unit Tests", () => {
+  // If you want to see what's happening, insert below render()
+  // screen.debug();
+  let onSubmit = jest.fn();
+  describe("Renders the Minimum Required Elements", () => {
```

Comment:
You could add a check for the privacy policy link since that is always rendered.

Reply:
Andrew added this already.